code/controllers/Processes/mob.dm

```diff
@@ -1,27 +1,34 @@
 /datum/controller/process/mob
-	var/tmp/datum/updateQueue/updateQueueInstance
 
 /datum/controller/process/mob/setup()
 	name = "mob"
 	schedule_interval = 20 // every 2 seconds
-	updateQueueInstance = new
+	start_delay = 16
 	if(!mob_master)
 		mob_master = new
 		mob_master.Setup()
 
 /datum/controller/process/mob/started()
 	..()
-	if(!updateQueueInstance)
-		if(!mob_list)
-			mob_list = list()
-		else if(mob_list.len)
-			updateQueueInstance = new
+	if(!mob_list)
+		mob_list = list()
+
+/datum/controller/process/mob/statProcess()
+	..()
+	stat(null, "[mob_list.len] mobs")
 
 /datum/controller/process/mob/doWork()
-	if(updateQueueInstance)
-		updateQueueInstance.init(mob_list, "Life")
-		updateQueueInstance.Run()
-		mob_master.process()
+	for(var/mob/M in mob_list)
```

Comment:
Shouldn't this have a spawn(-1) for each Life() call, so it behaves the same as the queue instance?